test/fixtures/multi/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const { resolve } = require('path');
+
+const { WebpackPluginServe } = require('../../../lib/');
+
+const serve = new WebpackPluginServe();
+
+module.exports = [
+  {
+    context: __dirname,
+    entry: ['./app.js', 'webpack-plugin-serve/client'],
+    mode: 'development',
+    output: {
+      filename: './dist-app.js',
+      path: resolve(__dirname, './output'),
+      publicPath: 'output/'
+    },
+    plugins: [serve],
+    resolve: {
+      alias: {
+        'webpack-plugin-serve/client': resolve(__dirname, '../../../client')
+      }
+    },
+    watch: true
+  },
+  {
+    context: __dirname,
+    entry: ['./worker.js', 'webpack-plugin-serve/client'],
+    mode: 'development',
+    output: {
+      filename: './dist-worker.js',
+      path: resolve(__dirname, './output'),
+      publicPath: 'output/'
+    },
+    plugins: [serve.attach()],
```

Comment:
Afaict this isn't documented, took me almost all day to find this, coupled with this bug I ran into here https://github.com/pmmmwh/react-refresh-webpack-plugin/issues/289 I was pretty stuck